### DISCUSSION TITLE
Is `sesame2` means a Sesame 3 device?

### DISCUSSION THREAD

**Discussion by takao-akiyoshi** [CANDY-HOUSE/SesameSDK_iOS_with_DemoApp#56] [Q&A]
I believe that both the new and old SDKs support SesameOS2 and above, and that "Sesame4" is defined separately, so I assume that this "Sesame2" refers to Sesame3.

https://github.com/CANDY-HOUSE/SesameSDK_iOS_with_DemoApp/blob/e8ebefea63ea03838908186049142edce29cb0c3/SesameSDK/Shared/Ble/CHBluetoothCenter.swift#L226

**Comment by Jerming** [comment]
Yes, you are correct. Sorry for making confused.
